VTUL/dlp-access
# add loading animation

**Add loading animation**
* * *

**JIRA Ticket**: (https://webapps.es.vt.edu/jira/browse/LIBTD-2526) (:star:)

* Other Relevant Links (Meeting note, project page, related pull requests, etc.)

# What does this Pull Request do? (:star:)
Add loading animation

# What's the changes? (:star:)
* Add loading animation

# How should this be tested?
* Load site and check that you get the Loading message instead of the error message

# Additional Notes:
* branch: `LIBTD-2526`

# Interested parties
@yinlinchen 

(:star:) Required fields
